be/src/agent/task_worker_pool.cpp

```diff
@@ -1900,6 +1900,7 @@ void CloneTaskPool::_clone_worker_thread_callback() {
         finish_task_request.__set_task_type(agent_task_req.task_type);
         finish_task_request.__set_signature(agent_task_req.signature);
         finish_task_request.__set_task_status(status.to_thrift());
+        finish_task_request.__set_report_version(_s_report_version);
```

Comment:
need to update _s_report_version when the return status is ok

Reply:
could you please explain why? According to the above discussion, I think, we should report the _s_report_version to fix the concurreny problem.

Reply:
I mean we need to ++ _s_report_version when the clone task in be success.
You can see other kind of task will ++ _s_report_version before report _s_report_version to Fe.

Reply:
> I mean we need to ++ _s_report_version when the clone task in be success. You can see other kind of task will ++ _s_report_version before report _s_report_version to Fe.

got it. Thanks a lot.